kaggle/loan-default-prediciton/metadata.yaml

```
@@ -0,0 +1,11 @@
+id:loan-defualt-prediction
+title:Loan Default Prediction - Imperial College London
+Constructing an optimal portfolio of loans
```

YAML 파일 포맷이 아닙니다.

```suggestion
title:Loan Default Prediction - Imperial College London Constructing an optimal portfolio of loans
```

위와 같이 변경해야 합니다.